sketch_oct9a.ino

```diff
@@ -1,4 +1,4 @@
-#if false
+#if true
```

Comment:
I'm confused why this was changed to 'true', and not the new file 'test_humidity'?

Reply:
I obviously had test_humidity true when uploading the code, but I think it makes the most sense for this file to have it on the master branch.

Reply:
sure, i gotcha